File: app/src/main/java/netpro/keyTransmitter/NormalKey.java
package netpro.keytransmitter;

public class NormalKey extends Key {

    private static final long serialVersionUID = 8877586457351207301L;

    public NormalKey(int columnSpan, int rowSpan, String description, Type type) {
        super(columnSpan, rowSpan, description, type);
    }

    @Override
    public void onActionUp() {
        send(keyCodeList);
    }
}

File: app/src/main/java/netpro/keytransmitter/NormalKey.kt
package netpro.keytransmitter

import android.view.MotionEvent
import android.view.View

/**
 * 指を離したときにキーを送信するボタンです
 */
class NormalKey(columnSpan: Int, rowSpan: Int, description: String, type: Key.Type) : Key(columnSpan, rowSpan, description, type) {

    override fun onActionUp(view: View, motionEvent: MotionEvent) {
        send(keyCodeList)
    }

    companion object {
        private val serialVersionUID = 8877586457351207301L
    }
}
